packages/main/src/plugin/index.ts

```diff
@@ -1321,6 +1332,22 @@ export class PluginSystem {
       },
     );
 
+    this.ipcHandle('app:update', async (): Promise<void> => {
+      await commandRegistry.executeCommand('update');
+    });
+
+    this.ipcHandle('app:update-available', async (): Promise<boolean> => {
+      return podmanDesktopUpdater.updateAvailable();
+    });
+
+    this.ipcHandle(
+      'app:get-release-notes',
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      async (): Promise<{ releaseNotesAvailable: boolean; notesURL: string; notes?: any }> => {
```

Comment:
here what is the goal of the optional any notes parameter as it is not used inside the function ?

Reply:
It's the notes information that is passed to the renderer side. If there are no release notes available, there is nothing to pass in `notes`, but if there are release notes available, they will be passed in the promise. I'll change the any type to the type of the release notes `{
  image: string;
  blog: string;
  title: string;
  summary: string;
}`